modules/base/shell.sh

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
 
-docker-compose -f ${DOCKER_COMPOSE_FILE} exec -e COLUMNS="`tput cols`" -e LINES="`tput lines`" -e SHELL=bash cli bash
\ No newline at end of file
+if [[ -z "$2" ]]; then
```

Comment:
This is a really basic check, maybe you know of a better way like with `checkParameters` or something?